src/backend/core/workflow_metrics.py

```
@@ -441,25 +462,62 @@ def calculate_crewai_cost(usage_metrics: dict, model_name: str = "gemini-2.0-fla
+        
+        if model_key in cost_map:
+            model_info = cost_map[model_key]
+            input_cost_per_token = model_info.get('input_cost_per_token', 0)
+            output_cost_per_token = model_info.get('output_cost_per_token', 0)
+            
+            cost = (
+                prompt_tokens * input_cost_per_token +
+                completion_tokens * output_cost_per_token
+            )
+            
+            logging.info(f"💰 CrewAI cost calculated from LiteLLM: ${cost:.6f} (model: {model_key})")
```

Ruff (0.14.10)

`info()` call on root logger

(LOG015)